Gems/Atom/Feature/Common/Code/Source/CoreLights/DirectionalLightFeatureProcessor.h

```diff
@@ -254,6 +254,11 @@ namespace AZ
             ShadowProperty& GetShadowProperty(LightHandle handle) { return m_shadowProperties.GetData(handle.GetIndex()); }
 
         private:
+
+            // This is currently fixed, but could be exposed to allow for user configuration
+            // See DirectionalLightShadowCalculator.azsli : DirectionalShadowCalculator::CalculateCascadeBlendAmount()
+            static constexpr const float CascadeBlendArea = 0.015;
```

Comment:
Worth being a cvar inside a cpp? That would allow runtime experimentation of different values..

Reply:
Worth being exposed all the way up to ui, but that comment was there before I just moved it. That's a separate piece of work